src/components/calcite-block/calcite-block.e2e.ts

```diff
@@ -28,6 +29,56 @@ describe("calcite-block", () => {
       </calcite-block>
   `));
 
+  it("it can be disabled", async () => {
+    const page = await setUpPage(
+      `
+        <calcite-block heading="heading" summary="summary" open collapsible>
+          <div class="content">content</div>
+        </calcite-block>
+    `,
+      { withPeerDependencies: true }
+    );
+
+    const content = await page.find(".content");
+    const clickSpy = await content.spyOnEvent("click");
+    await content.click();
+    expect(clickSpy).toHaveReceivedEventTimes(1);
+
+    const block = await page.find("calcite-block");
+    block.setProperty("disabled", true);
+    await page.waitForChanges();
+
+    await content.click();
+    expect(clickSpy).toHaveReceivedEventTimes(1);
+  });
+
+  it("has a loading state", async () => {
+    const page = await setUpPage(
+      `
+        <calcite-block heading="heading" summary="summary" open collapsible>
+          <div class="content">content</div>
+        </calcite-block>
+    `,
+      { withPeerDependencies: true }
+    );
+
+    expect(await page.find("calcite-block >>> calcite-loader")).toBeNull();
+
+    const content = await page.find(".content");
+    const clickSpy = await content.spyOnEvent("click");
+    await content.click();
+    expect(clickSpy).toHaveReceivedEventTimes(1);
+
+    const block = await page.find("calcite-block");
+    block.setProperty("loading", true);
+    await page.waitForChanges();
+
+    await content.click();
+    expect(clickSpy).toHaveReceivedEventTimes(1);
+
+    expect(await page.find("calcite-block >>> calcite-loader")).toBeTruthy();
+  });
```

Comment:
This is great imo. When it comes to tests sometimes its better to keep the code simple and not worry too much about it being DRY. Adding complexity can sometimes introduce bugs in the test code which is annoying.

Reply:
Agreed.